udata_front/theme/gouvfr/datagouv-components/src/components/AppLink/AppLink.stories.ts

```diff
@@ -0,0 +1,59 @@
+import type { Meta, StoryObj } from '@storybook/vue3';
+import { AppLink } from ".";
+import { vueRouter } from 'storybook-vue3-router';
+
+const meta = {
+  title: 'Components/AppLink',
+  component: AppLink,
+  tags: ['autodocs'],
+  parameters: {
+    docs: {
+      description: {
+        component: "The AppLink blablabla."
+      }
+    }
+  },
+} satisfies Meta<typeof AppLink>;
+
+export default meta;
+
+const args = {};
+
+export const SimpleExternalLink: StoryObj<typeof meta> = {
+  render: (args) => ({
+    components: { AppLink },
+    setup() {
+      return { args };
+    },
+    template: `<AppLink to="https://www.data.gouv.fr">
+                Link test
+              </AppLink>`,
+  }),
+  args,
+};
+
+export const SimpleInternalLinks: StoryObj<typeof meta> = {
```

Comment:
Can you put the `decorators` in the object declaration ? Like we do for render and args ? Or does it raise an error ?